### DISCUSSION TITLE
Ex-/Importing complete Database incl. Password History?

### DISCUSSION THREAD

**Discussion by TPISC** [keepassxreboot/keepassxc#6751] [Q&A] [answered]
So far I couldn't find a solution to ex- or import the password history. Is it somehow possible to import older passwords from a csv file? It would be much nicer that way, instead of having to import each record with the old password first, then have to modify each record's password.

**Comment by droidmonkey** [comment, marked as answer]
You could export/import the whole XML using the CLI tool. These functions aren't meant to be a backup/restore action for every feature of the database.
